torch/distributed/algorithms/model_averaging/averagers.py

```diff
@@ -98,6 +99,9 @@ def __init__(
                 "DistributedDataParallel should be used for this case."
             )
         self.period = period
+        # comm_memory_efficient is False, use a faster communication but more memory average strategy
```

Comment:
This data member can be moved to the base class, since it can be applied to all the model averaging algorithms.

Reply:
ok, i wil fix